pythnet/pythnet_sdk/Cargo.toml

```diff
@@ -17,7 +17,7 @@ solana-program = ["dep:solana-program", "dep:anchor-lang"]
 
 [dependencies]
 bincode = "1.3.1"
-borsh = "0.10.3"
+borsh = { version = "1.2.1", features = ["unstable__schema"] }
```

Comment:
you'll need to update the Solana target chain contract to use this version of borsh as well